### PR TITLE
Feature hum 7 create unit tests person service

### PR DESCRIPTION
**Pull request by SergioSousaDEV**
What this merge request do :
Create unit tests
Dependencyy Injection
Add MongoDb driver on presentation

What i need to focus on :
Dependency Injection
Unit tests

### CODE REVIEW

File: Presentation.WebApi/Startup.cs [hunk 1]
```diff
@@ -35,6 +39,24 @@ public void ConfigureServices(IServiceCollection services)
...
             services.AddSwaggerGenNewtonsoftSupport(); // explicit opt-in - needs to be placed after AddSwaggerGen()
 
+            var appSettings = new ApplicationSettings();
+
+            //Map AppSettings
+            Configuration.GetSection("App").Bind(appSettings);
+            services.AddSingleton<IApplicationSettings>(appSettings);
+
+            // Prepare MongoDB connection to provide to dependencies
+            var connectionString = appSettings.MongoDBSettings.ConnectionString;
```
Comment (naogomes) [inline Presentation.WebApi/Startup.cs:49]: Can we segregate this code in a private method to improve code reading ?

Comment (SergioSousaDEV) [reply Presentation.WebApi/Startup.cs:49]: I think so too, what is your sugestion?

File: Presentation.WebApi/Startup.cs [hunk 2]
```diff
@@ -35,6 +39,24 @@ public void ConfigureServices(IServiceCollection services)
...
+            var timeoutMs = appSettings.MongoDBSettings.TimeoutMs;
+
+            var databaseName = MongoUrl.Create(connectionString).DatabaseName;
+            var mongoDatabase = new MongoClient(connectionString).GetDatabase(databaseName);
+
+
+
+            var personRepository = new PersonRepository(mongoDatabase, timeoutMs);
```
Comment (naogomes) [inline Presentation.WebApi/Startup.cs:57]: Can we create a new class to do this dependency injection like ConfigureDependecies?

File: Presentation.WebApi/Startup.cs (hunk 1 above)
Comment (naogomes) [reply Presentation.WebApi/Startup.cs:49]: i'll make a call with you.

File: Application.Services.Tests/obj/Application.Services.Tests.csproj.nuget.dgspec.json [hunk 3]
```diff
@@ -4,6 +4,66 @@
     "C:\\Humanis\\HumanisApi\\Application.Services.Tests\\Application.Services.Tests.csproj": {}
```
Comment (naogomes) [inline Application.Services.Tests/obj/Application.Services.Tests.csproj.nuget.dgspec.json:4]: this file shouldn't be here. take care of your gitignore

File: Presentation.WebApi/Startup.cs (hunk 1 above)
Comment (SergioSousaDEV) [reply Presentation.WebApi/Startup.cs:49]: I appreciate your help!!!

Comment (naogomes) [reply Presentation.WebApi/Startup.cs:49]: are you available?

**Review by naogomes — APPROVED** [review]

File: Application.Services.Tests/obj/Application.Services.Tests.csproj.nuget.dgspec.json (hunk 3 above)
Comment (SergioSousaDEV) [reply Application.Services.Tests/obj/Application.Services.Tests.csproj.nuget.dgspec.json]: It is resolved in commit "Remove obj files from github".

**Pull request merged**